webapp/locales/de.json

```diff
@@ -830,6 +830,9 @@
   "user": {
     "avatar": {
       "submitted": "Erfolgreich hochgeladen!"
+    },
+    "profileHeader": {
+      "submitted": "Erfolgreich hochgeladen!"
     }
```

Comment:
May we can use one identifier for both cases if we show the same text?
like this:
```json
"image": {
  "submitted": "Erfolgreich hochgeladen!“
}
```

Or we do different texts as example:
- `Avatar-Bild erfolgreich hochgeladen!`
- `Profilbild erfolgreich hochgeladen!`

Reply:
I'll do the different texts one since it is better to letter the user know exactly what has changed to eliminate any confusion.